### DISCUSSION TITLE
Support Variations in Variable Fonts

### DISCUSSION THREAD

**Discussion by johnsicili** [webstudio-is/webstudio-community#76] [Ideas]
For [variable fonts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_fonts/Variable_fonts_guide), I'd like to be able to add customizations.

For example, instead of choosing between weight 300 and 400, I want to enter 350.

Here's what Webflow does:

![Screenshot 2023-11-26 at 5 55 03 PM](https://github.com/webstudio-is/webstudio-community/assets/93934362/ca517a5a-dba8-46ff-94be-3dbe02c465f8)

**Comment by oslogrolls** [comment]
This is what Firefox Developer Version offers. Sliders / scrubbing unlock the fun with VF.

![2024-04-16_23h26_07](https://github.com/webstudio-is/webstudio-community/assets/48279537/e9ba671c-2ece-4ac2-a7b0-e7664e7c5d66)
